au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/audio/VolumeTooltip.qml

```diff
@@ -62,8 +62,13 @@ StyledPopupView {
         }
     }
 
-    function show() {
-        openTimer.restart()
+    function show(noDelay = false) {
```

Comment:
Just minor nitpick about formatting to match overall qml style we have:
`        if (noDelay) {
            open();
        } else {
            openTimer.restart()
        }`